multinet/multinet/multinet.py

```diff
@@ -30,6 +30,9 @@ def graphql_query(query, variables=None):
             if excess > 0:
                 app.logger.error(f'{excess} more error{"s" if excess > 1 else ""}')
 
+            raise ValueError("Error when connecting to the DB.")
```

Comment:
I don't think `ValueError` is the right choice for this. `ValueError`s are meant for cases when the wrong range of values is presented to a function (like a square root function receiving a negative argument).

I'm also not sure it's necessary to raise an exception here in the first place: the `errors` value in the result will already flow back to the caller, which can then signal the appropriate status code and send the errors back to the client.

Reply:
Agreed. Remove the ValueError in defcb68